src/Generator/Program.fs

```diff
@@ -453,26 +483,87 @@ let formatParam (p: TypedParameterInfo) =
     let name = formatName p.name
     sprintf "%s %s" (p.typ |> typeToString) name
 
-let generateInterface enums (functions: TypedFunctionDeclaration[]) =
+let outputPath = "../binding/Binding"
+
+let generateDummyTypes () =
+    use backing = new StringWriter()
+    use writer = new System.CodeDom.Compiler.IndentedTextWriter(backing)
+    let writeLine (str: string) = writer.WriteLine str
+    let indent() = writer.Indent <- writer.Indent + 1
+    let unindent() = writer.Indent <- writer.Indent - 1
+    let usings = 
+        [
+            "System"
+        ]
+    for using in usings do
+        sprintf "using %s;" using
+        |> writeLine
+
+    writeLine ""
+    writeLine "namespace FooBar"
+    writeLine "{"
+    indent()
+    
+    let formatDummyType ty =
```

Comment:
In case `DummyType` has been renamed to `TypeName` or `TypeDescriptor` then consider changing it here too